src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -566,6 +567,8 @@ export default function MarkdownEditor({
   // The input element where the user inputs their comment.
   const input = useRef<HTMLTextAreaElement>(null);
 
+  const textWithoutMentionTags = useMemo(() => unwrapMentions(text), [text]);
```

Comment:
I considered doing this conditionally based on the `at_mentions` feature flag, but it should be harmless to do it undconditionally, and has the benefit that we avoid accidentally loading a mention tag in the text area.